README.md

```diff
@@ -397,3 +397,195 @@ const UserForm = () => (
   </div>
 );
 ```
+
+&nbsp;
+
+---
+
+&nbsp;
+
+## FAQ
```

Comment:
I think we could also stand to add a different section talking about how RRR doesn't care what shape your data is in. Maybe something about how whatever your Promise resolves with is what we store. What do you think?

I'm thinking that people may not immediately realize that they can just chain `.then`s forever and whatever they end up with will be used.

Reply:
Hrm sounds like two questions:

> Does my data need to be in a specific shape to use `react-remote-resource`?

> What are entries and how are entries created?

Reply:
I think that sounds right.